build/config/linux/libc.gni

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+declare_args() {
+  # The C library used by the compilation host.
+  # Expecting one of: "gnu" (glibc), "uclibc", "musl"
+  host_libc = ""
+
+  # The C library of the compilation target.
+  # Expecting one of: "gnu" (glibc), "uclibc", "musl"
+  target_libc = ""
+}
+
+if (host_libc == "") {
+  # XXX: actually detecting it is complicated
```

Comment:
In these cases, either file an issue and add a TODO referencing it here, or assume that it's unlikely to happen and document as something like "Detecting libc variant is difficult. Assume glibc unless informed otherwise."

Reply:
replaced with an actual detection script